packages/wonder-blocks-cell/src/components/internal/cell-core.js

```diff
@@ -80,6 +91,7 @@ const CellCore = (props: CellCoreProps): React.Node => {
 const styles = StyleSheet.create({
     wrapper: {
         background: Color.white,
+        color: Color.offBlack,
```

Comment:
Moving the color to the main wrapper in case we want to provide a new color via the `style` prop.